guides/v2.3/graphql/send-request.md

```diff
@@ -3,7 +3,25 @@ group: graphql
 title: GraphQL requests
 ---
 
-In Magento 2.3.1, you must send all queries and mutations as HTTP POST requests. 
+Magento GraphQL supports the HTTP GET and POST methods. You can send a query as a GET or POST request. GET requests can be cached, although Magento does not currently support this feature. Mutations must be POST requests.
```

Comment:
Please note, Magento does not have GET requests support within a scope of GraphQl API. This feature is merged and planned to be delivered in the future releases.